src/Services/privateApiService.js

```diff
@@ -42,4 +42,27 @@ export const getRequest =  async (url, id = null) => {
     }
 }
 
-
+export const PutRequest = async(endpoint,id,body) => {
+    try {
+        let res = await axios({
```

Comment:
Si después res no modifica su valor debería ser una constante, y en el método get se usa response como nombre, así que respetemos esa nomenclatura:

```suggestion
        const response = await axios({
```